sbol3/document.py

```diff
@@ -2,7 +2,6 @@
 import logging
 import warnings
 from typing import Dict, Callable, List, Optional
-
```

Comment:
We'd like to keep this separation between imports from core Python modules and imports from third party modules.

Reply:
OK, this is fixed now.  I never understood before what the rhyme or reason to how the imports were structured.